python/clean_content.py

```diff
@@ -121,7 +121,7 @@ def process_content_item(content_item, clean_content_writer, content_to_taxon_ma
     # Get content_id, taxon_id pairs and write to csv
     content_to_taxon_map_writer.writerows(map_content_id_to_taxon_id(content_item))
 
-    clean_content_writer.writerow(content_item.get(x) for x in HEADER_LIST)
+    clean_content_writer.writerow([content_item.get(x) for x in HEADER_LIST])
```

Comment:
It would be interesting to know what error this was (and to mention it in the commit message), as both using a generator and list work for me?